src/planscape/collaboration/permissions.py

```diff
@@ -139,3 +139,7 @@ def can_change(user: User, scenario: Scenario):
     @staticmethod
     def can_delete(user: User, scenario: Scenario):
         return is_creator(user, scenario.planning_area) or scenario.user.pk == user.pk
+
+    @staticmethod
+    def can_remove(user: User, scenario: Scenario):
```

Comment:
I'm not sure either. We currently have both in our codebase, somehow, and I think it might just be a mistake: the `CheckPermissionMixin` has `can_remove()`, but we also have `can_delete()` in a few places. I only just discovered this was causing problems in the parent permissions class, so this addition was an experiment, but I'll make a PR to consolidate these into `can_remove()`.